.azure-pipelines/apiview.yml

```diff
@@ -141,11 +141,8 @@ stages:
 
       - job: 'Test'
```

Comment:
I would really like to eventually see this under eng/pipelines or under the apiview directory as ci.yml there. I know this is only your backlog but every time I see it I think how inconsistent our tools repo is.